unit-tests/py/rspy/devices.py

```diff
@@ -223,7 +231,9 @@ def query( monitor_changes=True, hub_reset=False, recycle_ports=True ):
     #
     # Get all devices, and store by serial-number
     global _device_by_sn, _context, _port_to_sn
-    _context = rs.context( { 'dds': False } )
+    from rspy import test
```

Comment:
Not intended: `test` is for use in the actual tests and not for use here...
It can actually change the environment and may have side-effects.
Also, we're in a library here, inside a query() function -- we cannot make assumptions about our caller, or the existence of a "context".
Please use the `disable_dds` flag that I added, instead :).